functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/pages/execution/CommandPage.groovy

```diff
@@ -0,0 +1,63 @@
+package org.rundeck.tests.functional.selenium.pages.execution
+
+import groovy.transform.CompileStatic
+import org.openqa.selenium.By
+import org.openqa.selenium.WebElement
+import org.rundeck.tests.functional.selenium.pages.BasePage
+import org.rundeck.util.container.SeleniumContext
+
+/**
+ * Command page
+ */
+@CompileStatic
+class CommandPage extends BasePage {
+
+    String loadPath = ""
```

Comment:
this doesn't seem necessary

Reply:
This line replace the abstract method 'getLoadPath' to be implemented